internal/services/delegation_helpers.go

```diff
@@ -36,9 +36,7 @@ func (s *Service) registerUnbondingSpendNotification(
 		Index: 0, // unbonding tx has only 1 output
 	}
 
-	s.wg.Add(1)
```

Comment:
turned out no one calls for `s.wg.Wait()` which means `.Add(...)` and `.Done()` calls don't affect anything